tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -110,17 +110,43 @@ public static function assertCommandMatches(stdClass $expected, stdClass $actual
         static::assertDocumentsMatch($expected, $actual);
     }
 
+    /**
+     * @dataProvider provideTransactionsTests
+     * @group serverless
+     */
+    public function testTransactions(stdClass $test, ?array $runOn = null, array $data, ?string $databaseName = null, ?string $collectionName = null): void
+    {
+        $this->runTransactionTest($test, $runOn, $data, $databaseName, $collectionName);
+    }
+
+    public function provideTransactionsTests(): array
+    {
+        return $this->provideTests('transactions');
+    }
+
+    /**
+     * @dataProvider provideTransactionsConvenientApiTests
+     */
+    public function testTransactionsConvenientApi(stdClass $test, ?array $runOn = null, array $data, ?string $databaseName = null, ?string $collectionName = null): void
+    {
+        $this->runTransactionTest($test, $runOn, $data, $databaseName, $collectionName);
+    }
+
+    public function provideTransactionsConvenientApiTests(): array
+    {
+        return $this->provideTests('transactions-convenient-api');
+    }
```

Comment:
Were these methods necessary to split provisioning of all transaction tests? I assume convenient API tests weren't modified for serverless?